test/unit/concrete/comptroller/flash-fee/flashFee.t.sol

```diff
@@ -3,9 +3,23 @@ pragma solidity >=0.8.19 <0.9.0;
 
 import { UD60x18, ZERO } from "@prb/math/UD60x18.sol";
 
-import { Integration_Test } from "../../../Integration.t.sol";
+import { Base_Test } from "../../../../Base.t.sol";
+
+contract FlashFee_Unit_Concrete_Test is Base_Test {
+    /*//////////////////////////////////////////////////////////////////////////
+                                  SET-UP FUNCTION
+    //////////////////////////////////////////////////////////////////////////*/
+
+    function setUp() public virtual override {
+        Base_Test.setUp();
+
+        // Deploy V2 Core.
+        deployCoreConditionally();
```

Comment:
This is unfortunately too wasteful - it deploys all core contracts when just the comptroller is needed.

What we should do instead is:

- Write a new `Comptroller_Concrete_Test` contract in `test/unit/comptroller` (similar to [`NFTDescriptor`](https://github.com/sablier-labs/v2-core/blob/3df030516c7e9044742313c7cf17f15fdc1e9b05/test/unit/concrete/nft-descriptor/NFTDescriptor.t.sol#L26), which inherits from `Base_Test`
- Write a new `deployComptrollerConditionally` utility in that test contract
- Call the new deployer function in `setUp`
- Inherit from `Comptroller_Concrete_Test` instead of `Base_Test`

Reply:
Thanks for the clarification. Update the tests as mentioned above :+1:

Reply:
Thanks, @PraneshASP. I turned this into a `Comptroller_Unit_Shared_Test` contract so that the comptroller fuzz tests could be moved to `unit`, too.